scripts/update-version.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python3
+# This script will to help to manage rhpam components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
+#
+# Version pattern is: X.YY.Z
+# Dependencies:
+#  ruamel.yaml
+
+import argparse
+import os
+import pprint
+import re
+import sys
+
+from ruamel.yaml import YAML
+
+# All rhpam modules that will be updated.
+MODULES = {"businesscentral", "businesscentral-monitoring", "controller",
+           "dashbuilder", "kieserver", "process-migration",
+           "smartrouter"}
+# modules are: module_name/modules/module_name
```

Comment:
Maybe specify full path as given in the example?
(module_name/modules/module_name/module.yaml)
And specify that the version will be updated in these files?

Reply:
> I think the regex needs to be tuned a bit, right now when I tested the script with command `python scripts/update-version.py -v 7.10.1 --confirm`. I can see the value also changed in the ports section. I could see it in `git diff` in all module.yaml with value
> 
> ```
> ports:
>  - value: 7.10
> ```

Nice catch. updated.

Reply:
do you mean improve the comment before this env?